fastapi/dependencies/utils.py

```diff
@@ -261,6 +263,15 @@ def get_typed_signature(call: Callable[..., Any]) -> inspect.Signature:
     return typed_signature
 
 
+def is_coroutine_function(obj: Any) -> bool:
+    while isinstance(obj, functools.partial):
+        obj = obj.func
+
+    return asyncio.iscoroutinefunction(obj) or (
+        callable(obj) and asyncio.iscoroutinefunction(obj.__call__)
+    )
```

Comment:
https://github.com/encode/starlette/blob/858629f5188bc79d452600b1eb90eaa0045f6454/starlette/_utils.py#L1-L12

Reply:
![image](https://user-images.githubusercontent.com/13107421/196618691-065aa78a-2fdd-4074-aea1-fec239da1d9e.png)

Reply:
@Kludex

Reply:
yes, ignore it.